vmdb/app/models/ems_kubernetes.rb

```diff
@@ -0,0 +1,23 @@
+require 'kubeclient'
```

Comment:
Suggest moving this require into the method that actually needs it.  In this case `inventory` method.